client/src/index.jsx

```diff
@@ -15,20 +15,21 @@ import {
   NotFoundPage,
   DashboardPage,
   RecipeViewPage,
-} from './components/pages/Index.jsx';
+} from './components/pages';
 
-import './assets/css/font-awesome.css';
-import './assets/css/materialize.css';
-import './assets/css/style.css';
+import '../assets/js/jquery-3.2.1';
+import '../assets/js/materialize';
+
+import '../assets/css/font-awesome.css';
+import '../assets/css/materialize.css';
+import '../assets/css/style.scss';
 
 import reducers from './reducers';
 import actionTypes from './actions/actionTypes';
-import Authenticate from './utils/Authenticate.jsx';
+import Authenticate from './utils/Authenticate';
 import sampleRecipes from './utils/sampleRecipes';
 import setAuthorizationToken from './utils/setAuthorizationToken';
 
-import registerServiceWorker from './registerServiceWorker';
-
 // eslint-disable-next-line
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__();
```

Comment:
You don't need this on production ready code or set environment specific settings.